controller/src/main/java/com/flipkart/varadhi/controller/RetryPolicy.java

```diff
@@ -0,0 +1,23 @@
+package com.flipkart.varadhi.controller;
+
+import com.flipkart.varadhi.entities.cluster.OrderedOperation;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class RetryPolicy {
```

Comment:
can use builder with defaults here to allow easy construction of the instance

Reply:
Not doing this for now as not sure of final policy struct, also policy is created only once in code (excluding test). Would prefer to avoid test defaults in prod code.